### PR TITLE
Add a how-to.md in models

### PR DESCRIPTION
**Pull request by zuphilip**
There is quite some traffic to ocropy/models and I guess
that most of the people are looking for some existing models
to use.

### CODE REVIEW

File: models/how-to.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+You find the official two and more existing models listed in the wiki:
```
Comment (amitdo) [inline models/how-to.md]: >You find the official two and more existing models listed in the wiki: 
I suggest to change this to a form: 
>You can find the a list of available pre-trained models for ocropy here:

**Comment by amitdo** [comment]
* how-to.md
* How-to.md
* How-To.md
* HowTo.md
* HOWTO.md

???

**Comment by zuphilip** [comment]
About naming: I think only the `README.md` is shown directly in GitHub and therefore I changed that.

**Comment by zuphilip** [comment]
Thank you for the review @amitdo !

**Pull request merged**